### PR TITLE
ios support

### PR DESCRIPTION
**Pull request by futurepaul**
todo:
- [x] icon
- [x] figure out file save or just do it as a "share" action
- [x] github action for CI / CD
- [x] fix layout bugs
- [x] set up team apple account for signing
- [x] review all capacitor deps for any ios-specific gotchas

won't fix for now:
- :( splash screen 

a lot of this diff is autogenerated by capacitor, one notable thing is I'm changing the name of the barcode-scanner import so I get the right name for it in the podfile

also `MyCustomViewController.swift` is a cool place where we can do custom config stuff if we need to

### CODE REVIEW

File: src/root.css [hunk 1]
```diff
@@ -17,6 +16,10 @@ html {
     overscroll-behavior-y: none;
 }
 
+@media (prefers-color-scheme: light) {
+    /* we don't support this but I want the browser to know I care */
```
Comment (benthecarman) [inline src/root.css:20]: lol

**Comment by benthecarman** [comment]
The download logs/state stuff all seem to work, the only thing issue I see is if you cancel you get this little popup image

File: .github/workflows/ios-staging.yml [hunk 2]
```diff
@@ -0,0 +1,106 @@
+name: Release iOS Staging
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
```
Comment (benthecarman) [inline .github/workflows/ios-staging.yml:10]: before we merge this we probably only want this on `workflow_dispatch`, don't need a signet release for every pr and commit

File: package.json [hunk 3]
```diff
@@ -17,56 +17,58 @@
...
+        "tailwindcss": "^3.3.5",
+        "typescript": "^5.2.2",
+        "vite": "^4.5.0",
         "vite-plugin-pwa": "^0.14.7",
         "vite-plugin-wasm": "^3.2.2",
         "workbox-window": "^6.6.0"
     },
     "dependencies": {
-        "@capacitor/android": "^5.2.2",
+        "@capacitor-community/barcode-scanner": "npm:@mutinywallet/barcode-scanner@5.0.0-beta.3",
```
Comment (TonyGiorgio) [inline package.json:45]: This has been deprecated for this: https://github.com/capawesome-team/capacitor-mlkit/tree/main/packages/barcode-scanning

**Comment by benthecarman** [comment]
needs rebase

**Review by TonyGiorgio — APPROVED** [review]

**Review by benthecarman — APPROVED** [review]

**Pull request merged**